src/plugins/ni/nickel-1.1.0/src/hash.c

```diff
@@ -335,7 +335,7 @@ static uint32_t hashlittle (const void * restrict key, size_t length, uint32_t i
 	if (HASH_LITTLE_ENDIAN && ((u.i & 0x3) == 0))
 	{
 		const uint32_t * k = (const uint32_t *)key; /* read 32-bit chunks */
-#ifdef VALGRIND
+#if defined(VALGRIND) || defined(__SANITIZE_ADDRESS__)
```

Comment:
You can leave the fix as is, but the next time it would be better to define a single macro if one of the other macros are defined and use this. (Its quite likely that there are other memory check tools out there that again use other macros)